.github/workflows/style.yml

```
@@ -23,6 +23,10 @@ jobs:
         sudo apt-get install clang-format imagemagick ddnet-tools shellcheck pkg-config cmake ninja-build libfreetype6-dev libnotify-dev libsdl2-dev libsqlite3-dev libavcodec-dev libavformat-dev libavutil-dev libswresample-dev libswscale-dev libx264-dev python3-clang libvulkan-dev glslang-tools spirv-tools -y
         rustup default stable
         pip3 install pylint
+        arch=amd64
+        [ "$(uname -m)" = arm ] && arch=arm
+        sudo wget -O /usr/local/bin/shfmt https://github.com/mvdan/sh/releases/download/v3.8.0/shfmt_v3.8.0_$(uname | tr '[:upper:]' '[:lower:]')_$arch
```

```suggestion
        sudo wget -O /usr/local/bin/shfmt https://github.com/mvdan/sh/releases/download/v3.8.0/shfmt_v3.8.0_linux_amd64
```

We know our target architecture/system.

---

Do we? I wanted to make sure it does not break when github switches to arm runners. Or we switch it to also test on arm.

---

GitHub will not switch to arm runners. They'll also not switch to sparc runners, which this doesn't guard against. I'd like to not include unnecessary code.